src/bindings/c/include/openvino/c/ov_infer_request.h

```diff
@@ -69,16 +70,116 @@ typedef struct {
 OPENVINO_C_API(ov_status_e)
 ov_infer_request_set_tensor(ov_infer_request_t* infer_request, const char* tensor_name, const ov_tensor_t* tensor);
 
+/**
+ * @brief Sets an input/output tensor to infer on.
+ * @ingroup infer_request
+ * @param infer_request A pointer to the ov_infer_request_t.
+ * @param port  Port of the input or output tensor.
+ * @param tensor Reference to the tensor.
+ * @return Status code of the operation: OK(0) for success.
+ */
+OPENVINO_C_API(ov_status_e)
+ov_infer_request_set_tensor_by_node(ov_infer_request_t* infer_request, const ov_output_node_t* port, const ov_tensor_t* tensor);
```

Comment:
Which will be better? by_node or by_port or other else?

Reply:
ok, let use _by_port.

Reply:
done